Sming/Arch/Esp32/app.mk

```diff
@@ -11,7 +11,7 @@ LDFLAGS	+= \
 
 
 .PHONY: application
-application: $(CUSTOM_TARGETS) $(TARGET_BIN)
+application: $(TARGET_BIN)
```

Comment:
@mikee47 Why are you removing CUSTOM_TARGETS from an application? For example the SSL enabled applications still need it to create a private key and certificate.

Reply:
Ok, I see where it is used. Sorry, forget about this comment.